lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -63,10 +63,8 @@
 FEATURES_WITH_CERTS_DISABLED = settings.FEATURES.copy()
 FEATURES_WITH_CERTS_DISABLED['CERTIFICATES_HTML_VIEW'] = False
 
-FEATURES_WITH_CUSTOM_CERTS_ENABLED = {
-    "CUSTOM_CERTIFICATE_TEMPLATES_ENABLED": True
-}
-FEATURES_WITH_CUSTOM_CERTS_ENABLED.update(FEATURES_WITH_CERTS_ENABLED)
+FEATURES_WITH_CUSTOM_CERTS_ENABLED = FEATURES_WITH_CERTS_ENABLED.copy()
+FEATURES_WITH_CUSTOM_CERTS_ENABLED['CUSTOM_CERTIFICATE_TEMPLATES_ENABLED'] = True
```

Comment:
it looks like this is making the opposite of what was happening before, if `FEATURES_WITH_CERTS_ENABLED` had the `CUSTOM_CERTIFICATE_TEMPLATES_ENABLED` key, it will be overridden, and before it was the other way

Reply:
@eLRuLL previously `CUSTOM_CERTIFICATE_TEMPLATES_ENABLED ` was not present in `lms/envs/common.py` so it remained `True` after updating `FEATURES_WITH_CUSTOM_CERTS_ENABLED` with `FEATURES_WITH_CERTS_ENABLED` but now since we have added it to `lms/envs/common.py` and its default value is `False` so updating it will change it to `False` while we want it to be set to `True` for tests. So this change will keep the behaviour same as was before.